Traceback (most recent call last):
  File "/Users/raouf/handsontest/ai-hands-on-lab/.venv/lib/python3.9/site-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/Users/raouf/handsontest/ai-hands-on-lab/.venv/lib/python3.9/site-packages/nbclient/client.py", line 1314, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/Users/raouf/handsontest/ai-hands-on-lab/.venv/lib/python3.9/site-packages/jupyter_core/utils/__init__.py", line 165, in wrapped
    return loop.run_until_complete(inner)
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/asyncio/base_events.py", line 642, in run_until_complete
    return future.result()
  File "/Users/raouf/handsontest/ai-hands-on-lab/.venv/lib/python3.9/site-packages/nbclient/client.py", line 709, in async_execute
    await self.async_execute_cell(
  File "/Users/raouf/handsontest/ai-hands-on-lab/.venv/lib/python3.9/site-packages/nbclient/client.py", line 1062, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/Users/raouf/handsontest/ai-hands-on-lab/.venv/lib/python3.9/site-packages/nbclient/client.py", line 918, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
unique_sections = list(df_markdown_documents['solutionops_section'].unique())
color_mapping = {section: color for section, color in zip(unique_sections, colors)}
------------------


[0;31m---------------------------------------------------------------------------[0m
[0;31mTypeError[0m                                 Traceback (most recent call last)
Cell [0;32mIn[8], line 2[0m
[1;32m      1[0m unique_sections [38;5;241m=[39m [38;5;28mlist[39m(df_markdown_documents[[38;5;124m'[39m[38;5;124msolutionops_section[39m[38;5;124m'[39m][38;5;241m.[39munique())
[0;32m----> 2[0m color_mapping [38;5;241m=[39m {section: color [38;5;28;01mfor[39;00m section, color [38;5;129;01min[39;00m [38;5;28;43mzip[39;49m[43m([49m[43munique_sections[49m[43m,[49m[43m [49m[43mcolors[49m[43m)[49m}

[0;31mTypeError[0m: 'module' object is not iterable

